packages/connectivity/connectivity_macos/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.3
+
+* Adds an example app to trigger CI tests.
```

Comment:
You'll have to push and publish this before you can merge the changes to the core `connectivity` plugin.